Code/GraphMol/FileParsers/MolFileStereochem.cpp

```diff
@@ -615,6 +615,39 @@ void reapplyMolBlockWedging(ROMol &mol) {
   }
 }
 
+void clearMolBlockWedgingInfo(ROMol &mol) {
+  for (auto b : mol.bonds()) {
+    if (b->hasProp(common_properties::_MolFileBondStereo)) {
```

Comment:
You don't need to check if the property is there before you clear it.
Calling `clearProp()` on a non-existent property is a no-op. The documentation is wrong about this. :-(